src/typing_manager.ts

```diff
@@ -197,7 +237,7 @@ export class TypingManager implements Disposable {
             const matchedFirstKey = this.compositeMatchedFirstKey;
             this.compositeMatchedFirstKey = undefined;
             const { command, args } = this.compositeKeys[matchedFirstKey + key];
-            await commands.executeCommand(command, ...(args ? args : []));
+            commands.executeCommand(command, ...(args ? args : []));
```

Comment:
why remove `await`? same question for line 325

Reply:
As seen in the discussion of vscode related issues, make the execution as lightweight as possible. Whether or not await is added here has no effect on vscode-neovim.